src/main/java/edu/greenblitz/robotName/commands/CollectNote.java

```diff
@@ -11,9 +12,10 @@ public class CollectNote extends ConditionalCommand {
 
     public CollectNote() {
         super(
-                new NoteFromIntakeToShooter(),
```

Comment:
It seems logical edit, but it feels too much to ask you for such a think as logic.